schema-dex-amm-extended.graphql

```diff
@@ -167,9 +167,21 @@ interface Protocol {
   " Number of cumulative unique users "
   cumulativeUniqueUsers: Int!
 
+  " Number of cumulative liquidity providers "
```

Comment:
These 4 fields should also be in `DexAmmProtocol` which implements interface `Protocol`

Reply:
updated